modules/bot-improvement/src/views/full/components/FeedbackItem.tsx

```diff
@@ -58,51 +59,54 @@ const FeedbackItemComponent: FC<FeedbackItemComponentProps> = props => {
     await updateFeedbackItem({ status: 'pending' })
   }
 
-  const handleCorrectedActionTypeChange = async (correctedActionType: string) => {
+  const handleTypeChanged = async (correctedActionType: string) => {
     await updateFeedbackItem({
       correctedActionType,
       correctedObjectId: correctedActionType === 'qna' ? defaultQnaItemId : defaultGoalId
     })
   }
 
-  const handleCorrectedActionObjectIdChange = async (correctedObjectId: string) => {
+  const handleObjectIdChanged = async (correctedObjectId: string) => {
     await updateFeedbackItem({ correctedObjectId })
   }
 
+  const { correctedActionType, correctedObjectId, status, source } = feedbackItem
+
+  const actionItems = [
+    ...(correctedActionType === 'qna'
+      ? qnaItems.map(x => ({ label: x.data.questions[contentLang][0], value: x.id }))
+      : []),
+    ...(correctedActionType === 'start_goal' ? goals.map(x => ({ label: x.id, value: x.id })) : [])
```

Comment:
probably out of the scope of this PR but since we never finally added (nor plan to) the concept of goal can we remove all reference to them ?